pages.es/common/cp.md

```diff
@@ -0,0 +1,27 @@
+# cp
+
+> Copia archivos y directorios.
+
+- Copia un archivo a otra ruta:
+
+`cp {{ruta/hacia/archivo_original.ext}} {{ruta/hacia/archivo_copia.ext}}`
+
+- Copia un archivo a un directorio, manteniendo el nombre del archivo:
+
+`cp {{ruta/hacia/archivo_original.ext}} {{ruta/hacia/directorio_destino}}`
+
+- Copia de forma recursiva un directorio y su contenido a otra ruta (si la ruta de destino existe, el directorio se copiará dentro):
+
+`cp -R {{ruta/hacia/directorio_original}} {{ruta/hacia/directorio_copia}}`
+
+- Copia de forma recursiva y verbosa un directorio (muestra un listado de los archivos copiados):
+
+`cp -vR {{ruta/hacia/directorio_original}} {{ruta/hacia/directorio_copia}}`
+
+- Copia archivos de texto a otra ruta de forma interactiva (pregunta al usuario antes de sobreescribir):
+
+`cp -i {{*.txt}} {{ruta/hacia/directorio_destino}}`
+
+- Copia enlaces simbólicos sin mantener la referencia al original:
+
+`cp -L {{enlace}} {{ruta/hacia/directorio_destino}}`
```

Comment:
you forget the `--parents` example

Reply:
Oh, I see.
I have translated the `cp` page which is into the common directory.
I just noticed that there is another one into linux directory.

How do I proceed?
Maybe adding the `--parents` example in both?

Reply:
You can add that if you want, but is not needed for merge this PR.

Reply:
OK, merge it then 😃